action.yml

```
@@ -143,17 +142,12 @@ runs:
         echo "GIT_COMMITTER_NAME=${{ inputs.git-committer-name }}" >> $GITHUB_ENV
         echo "GIT_COMMITTER_EMAIL=<${{ inputs.git-committer-email }}>" >> $GITHUB_ENV
     - name: Run update-flake-lock.sh
-      run: $GITHUB_ACTION_PATH/update-flake-lock.sh
-      shell: bash
-      env:
-        GIT_AUTHOR_NAME: ${{ env.GIT_AUTHOR_NAME }}
-        GIT_AUTHOR_EMAIL: ${{ env.GIT_AUTHOR_EMAIL }}
-        GIT_COMMITTER_NAME: ${{ env.GIT_COMMITTER_NAME }}
-        GIT_COMMITTER_EMAIL: ${{ env.GIT_COMMITTER_EMAIL }}
-        NIX_OPTIONS: ${{ inputs.nix-options }}
-        TARGETS: ${{ inputs.inputs }}
-        COMMIT_MSG: ${{ inputs.commit-msg }}
-        PATH_TO_FLAKE_DIR: ${{ inputs.path-to-flake-dir }}
+      uses: DeterminateSystems/update-flake-lock/.github/actions/typescript@detsys-ts
```

```suggestion
      uses: DeterminateSystems/update-flake-lock/.github/actions/typescript@detsys-ts
```

Dicey, we should pin this to an exact hash.

---

Hm, why can't we rely on the repository-local file path? It should be possible to use `./.github/actions/typescript/action.yml` here. (We had to use `$GITHUB_ACTION_PATH` before because we were executing a file from this action's repository, but I think `uses` is evaluated in the context of the action itself before anything is actually run?)

---

I stopped being silly and remembered I could just run `node ./.../index.js`.